.github/workflows/delete-branch-after-merging.yml.OLD

```diff
@@ -0,0 +1,44 @@
+name: GitHub Actions - Delete head branch after a pull request is closed
```

Comment:
This file will not be used...
Keep it anyway?

Reply:
Yeah, I want to work on it in the following days. If it still is not necessary at all, I will delete it.

Reply:
Okay 👍